package.json

```diff
@@ -29,30 +29,30 @@
   ],
   "license": "MIT",
   "devDependencies": {
-    "broccoli-asset-rev": "^2.2.0",
-    "ember-cli": "^2.7.0",
+    "broccoli-asset-rev": "^2.4.2",
+    "ember-ajax": "^2.0.1",
+    "ember-cli": "2.7.0",
```

Comment:
When `ember-cli` upgrades a minor or patch things SHOULD never break based on semver. Sure things like `baseURL` should be replaced with `rootURL` but these are deprecations not hard requirements. If and when the build starts breaking because of this things can be addressed (it'll only impact when someone wants to update this repo anyways). This is my way to ensure when someone touches this repo in any way that the CI is running against the latest and greatest `ember-cli` and dying if the addon doesn't work with the latest and greatest.